keyboards/parallel/parallel_65/soldered/info.json

```diff
@@ -0,0 +1,467 @@
+{
+    "keyboard_name": "Parallel 65% PCB",
+    "maintainer": "matthewdias",
+    "layout_aliases": {
+        "LAYOUT_65_ansi_split_bs": "LAYOUT_65_ansi_blocker_split_bs",
+        "LAYOUT_65_ansi": "LAYOUT_65_ansi_blocker",
+        "LAYOUT_65_tsangan_split_bs": "LAYOUT_65_ansi_blocker_tsangan_split_bs",
+        "LAYOUT_65_iso": "LAYOUT_65_iso_blocker"
+    },
```

Comment:
```suggestion
```

Reply:
Confused what this change is for, will it not break the JSON format?

Reply:
Since this is a new board, there is no need to add layout aliases.